.github/workflows/ci.yaml

```diff
@@ -96,7 +96,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python: [ 3.6, 3.7, 3.8, 3.9 ]
```

Comment:
If we are testing against 3.10 any reason not to publish it?

Reply:
I don't really know. I limited myself to fixing the use of floats because it's subtle and I've seen builds get messy with python3.1 being used.

I think we could add 3.10 happily. But maybe we don't even need multiple versions of the endpoint being built anyway? This was all too much to noodle on when my goal was just "stop using 3.6 so that we can't use nice things". 😄